builtInFunctions/logsAndEvents.go

```diff
@@ -17,11 +18,18 @@ func addESDTEntryInVMOutput(vmOutput *vmcommon.VMOutput, identifier []byte, toke
 }
 
 func newEntryForESDT(identifier, tokenID []byte, nonce uint64, value *big.Int, args ...[]byte) *vmcommon.LogEntry {
-	nonceBig := big.NewInt(0).SetUint64(nonce)
+	tokenIdentifier := tokenID
```

Comment:
Should add a TODO and a task to remove all duplicated code regarding the token identifier